finalPrep/modZeroHero.js

```diff
@@ -0,0 +1,176 @@
+
+function assessSituation(dangerLevel, saveTheDay, badExcuse) {
+  console.log(assessSituation)
+}
+  var dangerLevel = 20
+  var saveTheDay = "I did it again!"
+  var badExcuse = "This is not good.....I'm gonna go home."
+
+
+// Your function should include an if/else statement that meets the following criteria
+//   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the badExcuse to the console
+//   - Anything dangerLevel that is between 10 and 50 should result in printing the saveTheDay string to the console
+//   - If the dangerLevel is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the console.
+
+if (dangerLevel > 50 ) {
+  console.log(badExcuse);
+} else if (dangerLevel >= 10 && dangerLevel < 50 ){
+  console.log(saveTheDay);
+} else{
+  console.log("Meh. Hard pass.")
+}
+
+//Test Cases
+var announcement = 'Never fear, the Courageous Curly Bracket is here!'
+var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
+// assessSituation(99, announcement, excuse) > Should print - 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
+//assessSituation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
+//assessSituation(3, announcement, excuse) > should print - "Meh. Hard pass."
+
+assessSituation(99, announcement, excuse)
+assessSituation(21, announcement, excuse)
+assessSituation(3, announcement, excuse)
+
+console.log('I think I forgot to lock up my 1992 Toyota Corolla. Be right back.')
+
+console.log('Never fear, the Courageous Curly Bracket is here!')
+
+console.log('Meh. Hard pass.')
+
+
+// Declare a new variable - scaryMonster - assigned to an Object with the following key/values
+//   - name (string)
+//   - smell (string)
+//   - weight (integer)
+//   - citiesDestroyed (array)
+//   - luckyNumbers (array)
+//   - address (object with following key/values: number , street , state, zip)
+
+var scaryMonster = {
+  name: "Rrrrrroomba",
+  smell: "cleanliness",
+  weight: 20,
+  citiesDestroyed: ["Living Room", "Kitchen", "Bedroom"],
+  luckyNumbers: [7, 8, 9, 10],
+  address: {
+    number: 1337,
+    street: "6th",
+    state: "Colorado",
+    zip: 80501
+  }
+}
+
+
+// Create a new class called SuperHero
+// - Your class should have the following DYNAMIC values
+//   - name
+//   - superpower
+//   - age
+// - Your class should have the following STATIC values
+//   - archNemesis, assigned to "The Syntax Error"
+//   - powerLevel = 100
+//   - energyLevel = 50
+class SuperHero {
+  constructor(name, superpower, age) {
+    this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+    this.archNemesis = "The Syntax Error"
+    this.powerLevel = 100
+    this.energyLevel = 50
+  }
+
+  sayName() {
+    console.log(this.name);
+  }
+
+  maximizeEnergy(){
+    this.energyLevel = 1000;
+  }
+
+  gainPower(){
```

Comment:
This `gainPower` should take an argument/parameter according to the instructions. For example:

```
gainPower(power) {
  this.powerLevel = this.powerLevel + power;
}
```